mailpoet/assets/js/src/segments/dynamic/types.ts

```diff
@@ -183,6 +183,7 @@ export interface EmailFormItem extends FormItem {
   opens?: string;
   emails?: string;
   clicks?: string;
+  timeframe?: Timeframe;
```

Comment:
Why was this change needed when we seem to keep the same TS version?

Reply:
I added it after the update because it was causing error with the updated TS version. I found that this prop was indeed missing in the definition but with the current TS version there is no error. I didn't revert the change as this is an improvement. So it was actually not needed it and I kept it kind of for the future. Sorry for the confusion.